### ISSUE TITLE
View definitions use different quotes for column aliases

### ISSUE THREAD

**Issue by gdsotirov** [gdsotirov/lemans24#2]
Definitions of views into the model (i.e. file `db/model/lemans24.mwb`) and SQL scripts (i.e. in folder `db/views/`) use different quotes for column aliases - either single quote (`'`), double quote (`"`) or back quote (`` ` ``). Not sure why I neglected this before, but it seems to lead to `Error Code: 1054. Unknown column 'Name' in 'field list'` when trying to create view `FemaleDrivers` with a script generated by MySQL Workbench in regards to #1 and the procedure to use the project. It seems to be due to the fact that views `AllDriversResults` and `FemaleDrivers` have different definitions :

```
CREATE TABLE IF NOT EXISTS `lemans24_new`.`FemaleDrivers` (`Name` INT, ...
...
CREATE TABLE IF NOT EXISTS `lemans24_new`.`AllDriversResults` (`"Name"` INT, ...
```

Note the additional double quotes in the definition of column `Name` of `AllDriversResults`. Also view `FemaleDrivers` selects specific columns from `AllDriversResults` instead of `SELECT *` in which case most probably the problem won't exist.

Also it's better to use `AS` explicitly with column aliases.

**Comment by gdsotirov** [comment]
Confirmed. After fixing column aliases only in views `FemaleDrivers ` and `AllDriversResults` the script generated by MySQL Workbench could now be executed successfully, because:

```
CREATE TABLE IF NOT EXISTS `lemans24_new`.`FemaleDrivers` (`Name` INT, ...
...
CREATE TABLE IF NOT EXISTS `lemans24_new`.`AllDriversResults` (`Name` INT,
```

Would have to update all views definitions.

**Comment by gdsotirov** [comment]
View definitions fixed with bd054f0, so close.
